src/containers/Login/LoginProviders.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2021 -present, NDLA
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { useEffect } from 'react';
+import { RouteComponentProps } from 'react-router';
+import { initializeFeideLogin } from '../../util/authHelpers';
+import { toHome } from '../../util/routeHelpers';
+
+interface Props {
+  authenticated: boolean;
+  authContextLoaded: boolean;
+  history: RouteComponentProps['history'];
+}
+
+export const LoginProviders = ({
+  authenticated,
+  history,
+  authContextLoaded,
+}: Props) => {
+  useEffect(() => {
+    if (authenticated && authContextLoaded) {
+      history.push(toHome());
```

Comment:
Støtter feide state-parameteren? Det brukes ofte for å hugse path før du trigger innlogging slik at du havner på samme sida som du sto før innlogginga. Vi gjør dette i ed, så kanskje vi kan gjøre tilsvarende her.

Reply:
Det er jeg relativt sikker på skal være mulig, det kan jeg ta å se på imorgen!